Tc dev 2021 1

narrucmot opened the pull request:
Devicetree changes for axi_intc settings and some other cleanup.

narrucmot committed:
UltraZed DTS changes

narrucmot committed:
Remove system.conf from UZ DTS files

narrucmot committed:
Merge branch '2021.1' of github.com:Avnet/meta-avnet into tc_dev_2021_1

narrucmot committed:
Updates for USB, etc. for 2021.1

wpatterson-witekio commented on recipes-bsp/device-tree/files/uz3eg-pciec/system-bsp.dtsi:
```diff
@@ -1,186 +1,199 @@
-	/* sata, usb3, dp */
-	clocks = <&gtr_clk1>, <&gtr_clk2>, <&gtr_clk3>;
-	clock-names = "ref1", "ref2", "ref3";
+   /* NC, SATA, USB3, DP */
```
Why is NC being added to the comment but there is not an additional clock added? i.e. why does the comment list 4 descriptions for 3 clocks? In other `psgtr` nodes there are 4 clocks, maybe the comment just got copy/pasted from one of those?

narrucmot replied:
The NC notes there is a No Connect.  There can be up to 4 GTR reference clocks described in the devicetree, but our hw platform only uses 3 and we note the 4th as NC.  This is what was recommended by the engineer at Xilinx.  The UZ7EV uses all 4 PS GTR reference clocks (though not connected the same to the Zynq MPSoC, and the 4th ref clock is used for PCIe).

wpatterson-witekio replied:
Ah, that makes sense

narrucmot committed:
Update amba_pl axi_intc dtsi settings

AlbertaBeef committed:
u96v2-sbc: Fix interrupts definition for axi_intc from <0 94 1> to <0 95 1>.

AlbertaBeef commented:
Approved, with fix for axi_intc device-tree definition ( b2e7f6b )
Verified with vitis build of dpu design for u96v2-sbc-base platform.

narrucmot committed:
Add axi_intc PS int connection description to devicetree for MZ and PZ

narrucmot committed:
Add description of axi_intc int connection to PS

narrucmot committed:
Add packagegroup with libdrm to zynqmp rootfs config

Libdrm includes modetest application, which is needed for the UltraZeds
and Ultra96-V2 to debug displayport settings, etc.
